pkg/render/node.go

```diff
@@ -393,6 +393,14 @@ func (c *nodeComponent) nodeRole() *rbacv1.ClusterRole {
 				Resources: []string{"blockaffinities"},
 				Verbs:     []string{"watch"},
 			},
+			{
+				// calico-upgrade service needs to query the calico-node
```

Comment:
I added the new RBAC to the calico-node role because the Calico for Windows installation uses the calico-node service account, and the new windows upgrade service uses it.

Reply:
So, unless I am missing some nuance, the windows upgrader daemonset should not be using the calico-node serviceaccount - it should have its own. The only reason might be if we use the serviceaccount credentials from the calico-for-windows upgrader for calico on the Windows nodes? I guess now that I think about it, that is probably the case.

Even so, we _probably_ should have our own serviceaccount and RBAC for the windows upgrader, even if it's 90% identical to calico/node. It's bad practice to provide unnecessary permissions to a service just because we didn't want to go through the effort of creating a separate service account.

Reply:
I agree we should have our own service account for Calico for Windows and we should fix this in the near future. After chatting with Song, we think we might not need this RBAC at all and can just verify the pod itself.